src/Compilers/Core/AnalyzerDriver/AnalyzerExceptionDescriptionBuilder.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.Analyzer
```

Comment:
Minor: We can probably just use Microsoft.CodeAnalysis.Diagnostics and avoid a new namespace.